### PR TITLE
feat: activate portfolio view

### PR DESCRIPTION
**Pull request by salimtb**
## **Description**

This PR turns on the Portfolio view feature flag.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/UI/Drawer/index.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const Drawer = forwardRef<DrawerRef, Props>((props, ref) => {
           chainId,
           networkOnboardingState,
         );
-        if (!networkOnboarded && !isPortfolioViewEnabled()) {
+        if (!networkOnboarded) {
```
Comment (sahar-fehri) [inline app/components/UI/Drawer/index.tsx:99]: Wonder why was this added here in the first place?
Maybe we want to remove the educational modal as part of the portfolio view feature?

Comment (salimtb) [reply app/components/UI/Drawer/index.tsx:99]: yes that's the goal , but i'm actually wondering if this should not be part of the unified list feature, wdyt ?

Comment (gambinish) [reply app/components/UI/Drawer/index.tsx:99]: I was wondering too. My thoughts were to get it green first, then remove/update tests as needed

Comment (gambinish) [reply app/components/UI/Drawer/index.tsx:96]: My intitial feeling is that we should keep it for now, uncless it's super buggy bc it touches a lot of e2e tests. If it's not super buggy like the extension was, then maybe we can ignore it's removal for now.

Comment (salimtb) [reply app/components/UI/Drawer/index.tsx:96]: this was a product decision

**Review by vinnyhoward — DISMISSED** [review]

File: app/util/networks/index.js [hunk 2]
```diff
@@ -497,5 +497,4 @@ export const isChainPermissionsFeatureEnabled =
 export const isPermissionsSettingsV1Enabled =
   process.env.MM_PERMISSIONS_SETTINGS_V1_ENABLED === 'true';
 
-export const isPortfolioViewEnabled = () =>
-  process.env.PORTFOLIO_VIEW === 'true';
+export const isPortfolioViewEnabled = () => true;
```
Comment (Cal-L) [inline app/util/networks/index.js:500]: This doesn't need to change since we do want to depend on the environment variable

**Review by Cal-L — COMMENTED** [review]
Left a comment

File: app/util/networks/index.js (hunk 2 above)
Comment (sahar-fehri) [reply app/util/networks/index.js]: Fixed here [be6c827](https://github.com/MetaMask/metamask-mobile/pull/12507/commits/be6c8277330a8994c0ab8310eb088d340f5ef407) 🙏

**Review by Cal-L — DISMISSED** [review]
Code LGTM

**Review by vinnyhoward — DISMISSED** [review]

**Review by Cal-L — APPROVED** [review]
LGTM

**Review by vinnyhoward — APPROVED** [review]

**Review by nickewansmith — APPROVED** [review]
